new-site/src/components/Playground.js

```diff
@@ -9,6 +9,58 @@ import theme from 'prism-react-renderer/themes/github';
 import './Playground.scss';
 import LiveErrorCore from './LiveErrorCore';
 
+const sanitizeConfig = {
+  allowedTags: false,
+  disallowedTagsMode: 'discard',
+  allowedAttributes: {
```

Comment:
How do we know which attributes should be allowed and which not? This seems little odd to do it this way. Would be maybe easier if we could list attributes we do not want 🤔

Reply:
I'm not sure is it possible: https://github.com/apostrophecms/sanitize-html#readme